### ISSUE TITLE
Display dimension of the current cell

### ISSUE THREAD

**Issue by turion** [jamievicary/globular#57]
It would be very handy if there was a number, just on top of the "Project" selector, that displays the dimension of the current cell. That way I can be sure e.g. that I've actually pressed the "Identity" button often enough. I'm sure that should be easy to implement.

**Comment by jamievicary** [comment]
You can work it out as

  Dimension = (Dimension of image) + (Current project value) + (Number of
slice controls)

Of course, this is not to say adding it explicitly wouldn't be useful.

**Comment by turion** [comment]
What's "Dimension of image"? Sort of "when I see braids, it's 3-dimensional"? That seems useful, but I agree that an explicit dimension counter would be more useful, especially for beginners.

**Comment by jamievicary** [comment]
No, I meant something much simpler: when you're looking at a picture in
Globular, it's either 2d, 1d or 0d. For example, a picture of some braids
is a 2d image. If you increase the "Project" counter, you can force
Globular to draw a 1d or 0d image of your composite.
